src/mos_api.asm

```diff
@@ -1579,6 +1580,29 @@ $$:			PUSH	DE		; int length
 			POP	DE
 			RET
 
+; Clear VDP flag(s)
+; C: bitmask of flags to clear
+; Returns:
+; - A: vdp flags
+mos_api_clear_vdp_flags:
+			PUSH	HL		; Save HL
+			LD	HL, _vpd_protocol_flags
+			LD	A, C
+			CPL			; Invert the bitmask
+			AND	(HL)		; Clear the requested flags
+			LD	(HL), A		; Save the new flags
+			POP	HL
+			RET
+
+; Wait until VDP flag(s) are set, or timeout
+; C: bitmask of flags to wait for
+; Returns:
+; - A = status code (0 = OK, 15 = timeout (FR_TIMEOUT))
+mos_api_wait_vdp_flags:
+			PUSH	BC

```

Comment:
Is this missing a corresponding pop?

Reply:
doh!  yes it is.  thanks for spotting that!